test/Parse/errors.swift

```diff
@@ -110,10 +110,18 @@ func postThrows() -> Int throws { // expected-error{{'throws' may only occur bef
   return 5
 }
 
+func postThrows2() -> throws Int { // expected-error{{'throws' may only occur before '->'}}{{20-22=throws}}{{23-29=->}}
+  return 5
```

Comment:
Can you test that we've recovered as a throwing function as well, by calling something else that throws and not getting a warning?